coderd/database/queries/notifications.sql

```diff
@@ -174,3 +174,23 @@ SELECT *
 FROM notification_templates
 WHERE kind = @kind::notification_template_kind
 ORDER BY name ASC;
+
+-- name: GetReportGeneratorLogByUserAndTemplate :one
```

Comment:
Nit: these functions should all be namespaced under "notifications" since it's a generic term.

```suggestion
-- name: GetNotificationReportGeneratorLogByUserAndTemplate :one
```

Reply:
My point here is that these reports are only used for notifications; "report" is such a generic term, and having these in the global scope along with all other queries will be confusing without being scoped to notifications.

Reply:
Ok, will rename the package and relevant structures 👍